### PR TITLE
Minor Improvements

### PR DESCRIPTION
**Pull request by arturobernalg**
* Remove redundant initializer
*  Use Empty array

### CODE REVIEW

File: src/main/java/org/apache/commons/text/StrTokenizer.java [hunk 1]
```diff
@@ -124,10 +124,12 @@
     private StrMatcher trimmerMatcher = StrMatcher.noneMatcher();
 
     /** Whether to return empty tokens as null. */
-    private boolean emptyAsNull = false;
+    private boolean emptyAsNull;
     /** Whether to ignore empty tokens. */
     private boolean ignoreEmptyTokens = true;
 
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/text/StrTokenizer.java:131]: -1, reuse ArrayUtils.

File: src/main/java/org/apache/commons/text/StringTokenizer.java [hunk 2]
```diff
@@ -130,10 +130,12 @@
     private StringMatcher trimmerMatcher = StringMatcherFactory.INSTANCE.noneMatcher();
 
     /** Whether to return empty tokens as null. */
-    private boolean emptyAsNull = false;
+    private boolean emptyAsNull;
     /** Whether to ignore empty tokens. */
     private boolean ignoreEmptyTokens = true;
 
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/text/StringTokenizer.java:137]: -1, reuse ArrayUtils.

**Review by garydgregory — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/commons/text/StrTokenizer.java (hunk 1 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/text/StrTokenizer.java]: True. Change it for ArrayUtils.EMPTY_STRING_ARRAY

File: src/main/java/org/apache/commons/text/StringTokenizer.java (hunk 2 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/text/StringTokenizer.java]: Changed  for ArrayUtils.EMPTY_STRING_ARRAY

**Pull request merged**